### PR TITLE
Added example to to create facility

### PR DESCRIPTION
**Pull request by liskaj**

### CODE REVIEW

File: common/tandemClient.js [hunk 1]
```diff
@@ -628,6 +786,23 @@ export class TandemClient {
         return data;
     }
 
+    /**
+     * Imports model for given facility. The model must be created using {@link createModel} function.
+     * @param {string} facilityId 
+     * @param {object} inputs
```
Comment (bakharal) [inline common/tandemClient.js:807]: Would it make sense to define a JSDoc type here?

File: facility/create-facility.js [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+        roomAssignment: true,
+        conversionMethod: 'v3'
+    });
+    // STEP 6 - wait for import to finish
+    let modelProps;
+
+    while (true) {
+        modelProps = await client.getModelProps(modelResult.modelId);
+
+        if ((modelProps.state.state === ModelState.Ready)  || (modelProps.state.state === ModelState.Failed)) {
```
Comment (bakharal) [inline facility/create-facility.js:115]: I would not do polling here. This is an async API and user can use model ID to query task status. If you really want to show a full example with waiting here, I would at least guard infinite loop with timeout.

File: facility/create-facility.js [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+        param.applicationFilters = {
+            [filterKey]: [...param.applicationFilters].sort()
+        };
+        params.push(param);
+        return params;
+    }, []).sort((a, b) => a.uuid.localeCompare(b.uuid, 'en'));
+
+    const result = {
+        ...template,
+        library: true,
```
Comment (bakharal) [inline facility/create-facility.js]: How could a generated template be from the library?

**Review by bakharal — COMMENTED** [review]
I get the intent here, but I would rather encourage user to curate template via Tandem UI, where we have safeguards/hints/guides/best practices available to them. Current approach looks a bit confusing IMO, since there is no way customers will really generate templates automatically ATM - it's just too complex for real-world scenarios.

File: common/tandemClient.js (hunk 1 above)
Comment (liskaj) [reply common/tandemClient.js:807]: Done

**Comment by liskaj** [comment]
@bakharal The example doesn't create new template. It selects one using name and then combines template data with classification and parameters - this is because this is needed for POST twins/:facilityId/template call. Same logic is used by our UI.
The one thing is that I use JSON files from our app servers instead of fetching them - because endpoint is version specific.

**Review by bakharal — APPROVED** [review]

**Pull request merged**